### PR TITLE
Handle Runner init errors in CSIController and Clients.

### PR DESCRIPTION
**Pull request by pietrzakacper**
Now when Runner fails on start because for example the sequence path is incorrect, it will exit with the appropriate exit code, then this exit code will be mapped in csi controller to error message and sent out in REST API response to sequence start request

### Verify
Make sure you have the CLI installed from this branch and STH running.
1. Modify package.json of [some sample](https://github.com/scramjetorg/scramjet-cloud-docs/tree/main/samples/hello) to an invalid one
2. Pack, send and start the sequence
```
si pack . -o dist.tar.gz && si seq send dist.tar.gz && si seq start -
```
3. You should see an error in CLI similar to this:
image

### CODE REVIEW

File: packages/runner/src/bin/start-runner.ts [hunk 1]
```diff
@@ -12,22 +13,22 @@ const instanceId = process.env.INSTANCE_ID;
 
 if (!instancesServerPort || instancesServerPort !== parseInt(instancesServerPort, 10).toString()) {
     console.error("Incorrect run argument: instancesServerPort");
-    process.exit(1);
+    process.exit(RunnerExitCode.INVALID_ENV_VARS);
```
Comment (MichalCz) [inline packages/runner/src/bin/start-runner.ts:16]: @pietrzakacper providing exit code this way is deprecated, it should be done via `process.exitCode = `

Comment (pietrzakacper) [reply packages/runner/src/bin/start-runner.ts:16]: I dont think it's deprecated. Setting a property is just a way for setting exitCode when we don't want to exit forcefully. https://nodejs.org/api/process.html#processexitcode

Comment (pietrzakacper) [reply packages/runner/src/bin/start-runner.ts:16]: In this case, we want to

Comment (MichalCz) [reply packages/runner/src/bin/start-runner.ts:16]: Oki. I was mistaken then. :)

**Review by f1ames — APPROVED** [review]
LGTM :+1: 

---

Tested with:

```
yarn start --runtime-adapter process
```

and then sending sequence with invalid `main` field in `package.json`.

Via `si`:


image


Via API:


image

---

@pietrzakacper from what I understand it's only about invalid `main` field? I tried with differently broken sequences (empty `index.js`, syntax error, invalid tar) but the errors then are in different places (before runner initialization or after runner is started) so it's not in the scope of this PR right?

**Review by MichalCz — APPROVED** [review]

**Pull request merged**